packages/nektar/package.py

```diff
@@ -32,9 +32,22 @@ class Nektar(CMakePackage):
     variant("tinyxml", default=True, description="Builds with external tinyxml support")
     variant("hdf5", default=False, description="Builds with hdf5 support (conflicts with builtin?)")
     variant("scotch", default=True, description="Builds with scotch partitioning support")
-    variant("demos", default=True, description="Build demonstration codes")
-    variant("solvers", default=True, description="Build example solvers")
+    variant("demos", default=False, description="Build demonstration codes")
     variant("python", default=True, description="Enable python support")
+    # Solver variants; all but CompressibleFlow default to disabled
+    variant("acoustic_solver", default=False, description="Builds a library and an executable associated with the Acoustic solver")
+    variant("adr_solver", default=False, description="Builds a library and an executable associated with the ADR solver")
+    variant("cardiac_solver", default=False, description="Builds a library and an executable associated with the Cardiac electrophysiology solver")
+    variant("compflow_solver", default=True, description="Builds a library and an executable associated with the CompressibleFlow solver")
```

Comment:
Would `compflow_solver` be needed by most users, or did you just make it default to `True` because it will be needed by `NESO`? If the latter, there's really no need since we can explicitly specify it in the dependency for the NESO package.

Reply:
> Would `compflow_solver` be needed by most users, or did you just make it default to `True` because it will be needed by `NESO`? If the latter, there's really no need since we can explicitly specify it in the dependency for the NESO package.

It was the latter - I'll set it back to False when I add the dependency.

Reply:
Done.